### PR TITLE
Fixed criticality score

### PR DESCRIPTION
**Pull request by Cheshiriks**
Fix criticality score

### CODE REVIEW

**Review by akuleshov7 — APPROVED** [review]
Why we don’t have criticality in OSV data please check!

**Comment by Cheshiriks** [comment]
> Why we don’t have criticality in OSV data please check!

Yes, in all files the severityNum is null
Снимок экрана 2023-10-12 в 12 50 27

File: save-cosv/src/main/kotlin/com/saveourtool/save/cosv/service/VulnerabilityMetadataService.kt [hunk 1]
```diff
@@ -145,9 +145,7 @@ class VulnerabilityMetadataService(
         ): VulnerabilityMetadata = apply {
             summary = entry.summary ?: entry.details?.take(SUMMARY_LENGTH_FROM_DETAILS)?.let { "$it ..." } ?: "Summary not provided"
             details = entry.details ?: "Details not provided"
-            severityNum = entry.severity?.firstOrNull()
-                ?.scoreNum
-                ?.toFloat() ?: 0f
+            severityNum = entry.severity?.firstOrNull()?.let { getScore(it) } ?: 0f
```
Comment (nulls) [inline save-cosv/src/main/kotlin/com/saveourtool/save/cosv/service/VulnerabilityMetadataService.kt:148]:
```suggestion
            severityNum = entry.severity?.firstOrNull()?.let { it.scoreNum?.toFloat() ?: getScore(it) } ?: 0f
```

**Comment by nulls** [comment]
> Why we don’t have criticality in OSV data please check!

it's a new field in COSV, not in OSV

File: save-cosv/src/main/kotlin/com/saveourtool/save/cosv/service/VulnerabilityMetadataService.kt (hunk 1 above)
Comment (Cheshiriks) [reply save-cosv/src/main/kotlin/com/saveourtool/save/cosv/service/VulnerabilityMetadataService.kt:148]: It is not necessary. We decided to always calculate the value from the vector and not take it from the string. But we compare them and log them if they do not match.

**Pull request merged**